src/OpenTelemetry/Trace/Export/Abstractions/ITraceComponent.cs

```diff
@@ -0,0 +1,31 @@
+﻿// <copyright file="ITraceComponent.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Trace
+{
+    using OpenTelemetry.Trace.Export;
+
+    /// <summary>
+    /// Trace component holds all the extensibility points required for distributed tracing.
+    /// </summary>
+    public interface ITraceComponent : ITraceComponentBase
```

Comment:
same concern here: why we need 2 interfaces and both are in Abstractions?

Reply:
It will get away anyway: https://github.com/open-telemetry/opentelemetry-dotnet/issues/18

Reply:
Second layer has been removed as described above.